### ISSUE TITLE
food!

### ISSUE THREAD

**Issue by DoctorDiLauro** [DoctorDiLauro/insect-injector#1]
@thomasjbradley

For the most part I understand what is happening in the code, but I'm not sure how to use eatsInsects/eatsDung/eatsPlants to get the code to execute the if statement. How are these used in my object that I am creating?

**Comment by thomasjbradley** [comment]
Yes, I see you're really close. Looking at the output function you can see `food.eatsInsects`—that `.` means that `food` is what kind of thing?

**Comment by DoctorDiLauro** [comment]
Object in an object in an array...sneaky. Got it, thanks!
